src/en/qiscans/src/eu/kanade/tachiyomi/extension/en/qiscans/QiScans.kt

```diff
@@ -0,0 +1,127 @@
+package eu.kanade.tachiyomi.extension.en.qiscans
+
+import eu.kanade.tachiyomi.multisrc.iken.GenreFilter
+import eu.kanade.tachiyomi.multisrc.iken.Iken
+import eu.kanade.tachiyomi.multisrc.iken.SelectFilter
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.network.interceptor.rateLimit
+import eu.kanade.tachiyomi.source.model.Filter
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import keiyoushi.utils.parseAs
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Request
+import okhttp3.Response
+import rx.Observable
+import rx.schedulers.Schedulers
+import java.util.concurrent.TimeUnit
+
+class QiScans : Iken(
+    "Qi Scans",
+    "en",
+    "https://qiscans.org",
+    "https:///api.qiscans.org",
+) {
+
+    override val client = super.client.newBuilder()
+        .rateLimit(3, 1, TimeUnit.SECONDS)
```

Comment:
btw do we need it that strict?

Reply:
`rateLimit(1)` should be ok. again please test that with a large library update. (you can browse the source on fresh install of tachiSy then add all db manga to library from debug settings).